robottelo/config/casts.py

```diff
@@ -70,3 +70,30 @@ class Tuple(List):
     """
     def __call__(self, value):
         return tuple(super(Tuple, self).__call__(value))
+
+
+class Dict(List):
+    """Cast a comma separated list of key=value to a dict.
+
+    :param str value: A comma separated string to cast to a dict.
+    """
+    def __call__(self, value):
+        return dict(v.split('=') for v in super(Dict, self).__call__(value))
+
+
+class WebdriverDesiredCapabilities(Dict):
+    """Cast a comma separated list of key=value to a
+    webdriver.DesiredCapabilities dict.
+
+    :param str value: A comma separated string to cast to a
+        webdriver.DesiredCapabilities dict.
+    """
+    def __call__(self, value):
+        desired_capabilities = super(
+            WebdriverDesiredCapabilities, self).__call__(value)
+        for k, v in desired_capabilities.items():
+            if v.lower() == 'true':
+                desired_capabilities[k] = True
+            elif v.lower() == 'false':
```

Comment:
I really don't see reason to change the code, using an explicit else, will make the code more verbose and will not add any meaning since `else: pass` needs two lines and will not add anything than saying that else is not needed, not adding else say the same.

For the approach @rochacbruno suggested I like it more, but I fell that there is no need to run lower two times.

Decided to add a comment instead in order to make it more clear and avoid not necessary function calls and statements.

If you really want me to change this, let me know and I change.